shared/graphql/schema.graphql

```diff
@@ -556,6 +567,9 @@ type Query {
   searchMorePackages(after: String!, size: Int = 30): PackagesSearchMoreResult!
   subscription: SubscriptionState!
 
+  bucketAccessCounts(bucket: String!, window: Int!): BucketAccessCounts
+  objectAccessCounts(bucket: String!, key: String!, window: Int!): AccessCounts
```

Comment:
we don't have a proper "bucket" entry point currently, and i don't feel comfortable designing / adding that now, so let's just add separate queries.
tho feel free to add a simple container type to group this if you like